src/main/resources/view/MainWindow.fxml

```diff
@@ -35,47 +35,39 @@
+          <VBox styleClass="stack-pane" HBox.hgrow="ALWAYS">
+            <StackPane styleClass="stack-pane">
+              <padding>
+                <Insets top="10" />
+              </padding>
+              <ImageView fx:id="logoImage" fitWidth="200.0" pickOnBounds="true" preserveRatio="true"/>
+            </StackPane>
+            <StackPane fx:id="statisticsPlaceholder" prefWidth="200.0" maxHeight="100.0" styleClass="card"/>
+            <StackPane fx:id="rsvpPlaceholder" prefHeight="100.0" prefWidth="200.0" maxHeight="100.0" styleClass="card" />
+            <StackPane fx:id="dietaryPlaceholder" prefHeight="200.0" prefWidth="200.0" styleClass="card" />
           </VBox>
-          <VBox fx:id="guestList" prefWidth="350" styleClass="stack-pane" VBox.vgrow="ALWAYS">
-            <padding>
-              <Insets bottom="10" left="10" right="10" top="10" />
-            </padding>
+          <VBox fx:id="guestList" prefWidth="350" styleClass="stack-pane" VBox.vgrow="ALWAYS" HBox.hgrow="ALWAYS">
             <StackPane fx:id="guestListPanelPlaceholder" VBox.vgrow="ALWAYS" />
           </VBox>
-          <VBox fx:id="vendorList" prefWidth="350" styleClass="stack-pane" VBox.vgrow="ALWAYS">
-            <padding>
-              <Insets bottom="10" left="10" right="10" top="10" />
-            </padding>
+          <VBox fx:id="vendorList" prefWidth="350" styleClass="stack-pane" VBox.vgrow="ALWAYS" HBox.hgrow="ALWAYS">
             <StackPane fx:id="vendorListPanelPlaceholder" VBox.vgrow="ALWAYS" />
           </VBox>
         </HBox>
 
         <HBox>
-
-          <VBox minWidth="400.0" prefWidth="850.0">
-            <VBox>
-              <StackPane fx:id="resultDisplayPlaceholder" maxHeight="200" minHeight="100" prefHeight="100" styleClass="pane-with-border" VBox.vgrow="NEVER">
-                <padding>
-                  <Insets bottom="5" left="10" right="10" top="5" />
-                </padding>
-              </StackPane>
-              <StackPane fx:id="commandBoxPlaceholder" styleClass="stack-pane" VBox.vgrow="NEVER">
-                <padding>
-                  <Insets bottom="5" left="10" right="10" top="5" />
-                </padding>
-              </StackPane>
-            </VBox>
+          <VBox minWidth="400.0" prefWidth="10000" style="-fx-background-color: #f1e5e5">
+            <StackPane fx:id="resultDisplayPlaceholder" minHeight="60" maxHeight="100.0" styleClass="card" VBox.vgrow="NEVER">
+              <padding>
+                <Insets bottom="5" left="10" right="10" top="5" />
+              </padding>
+            </StackPane>
+            <StackPane fx:id="commandBoxPlaceholder" styleClass="command-box" VBox.vgrow="NEVER">
+              <padding>
+                <Insets bottom="10" left="10" right="10" top="10" />
+              </padding>
+            </StackPane>
           </VBox>
-
-
-
         </HBox>
 
         <StackPane fx:id="statusbarPlaceholder" prefWidth="405.0" VBox.vgrow="NEVER" />
```

Comment:
Leaving the comment here as I'm not sure what's causing the issue. The application doesn't scale with size.
![image](https://github.com/AY2324S1-CS2103T-F11-2/tp/assets/99188356/8560eac2-7700-4385-b375-005858faf9d0)